shared/local-debug.native.js

```diff
@@ -8,6 +8,8 @@ import {updateDebugConfig} from './actions/dev'
 
 const nativeBridge = NativeModules.KeybaseEngine
 
+const PERF = false
```

Comment:
Will you always want `__DEV__ === false` in addition to this when using it to get some react-native optimizations?